source/appModules/webconferenceplugin.py

```diff
@@ -0,0 +1,12 @@
+import appModuleHandler
+from NVDAObjects.behaviors import ProgressBar
+import controlTypes
+import ui
```

Comment:
During merging, we could just remove extra imports and add headers (q trivial thing that can be done by Jamie or others). Thanks, looks fine to me.